modules/velero-obtaining-by-downloading.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="velero-obtaining-by-downloading_{context}"]
+= Downloading the Velero CLI tool
+
+You can download and install the Velero CLI tool by following the instructions on the link:https://velero.io/docs/v1.8/basic-install/#install-the-cli[Veleo documentation page].
+
+The page includes instructions for:
+
+* MacOS using Homebrew
+* GitHub
+* Windows using Chocolatey
+
+.Prerequisites
+
+* Access to a Kubernetes cluster, v1.16 or later, with DNS and container networking enabled.
+* `kubectl` installed locally.
+
+.Procedure
```

Comment:
I see your point, but I'd rather leave it as a module in an assembly than a few lines of text in an assembly because if we ever need to expand it (possible) or use this text elsewhere (admittedly unlikely), it will be easier if it's already a module. Thanks for giving me the option.